uniflow/flow/flow.py

```diff
@@ -3,16 +3,21 @@
 import logging
 from typing import Any, Mapping, Sequence
 
-from uniflow.flow import constants
+from uniflow import constants
 from uniflow.flow.flow_factory import FlowFactory
 from uniflow.node.node import Node
 from uniflow.op.basic.linear_op import LinearOp
 
+logger = logging.getLogger(__name__)
+
 
 class Flow:
     """Flow class."""
 
+    tag = constants.BASIC
```

Comment:
nit: do we really need to tag this base flow? We should only tag flows registered into the flow factory.